File: codeforces/379A/py/379a.py
import sys

a, b = map(int, sys.stdin.readline().split())

hour = 0
wentout = 0
while a > 0:
    hour += a
    wentout += a

    if wentout >= b:
        a = int(wentout / b)
        wentout -= a*b
    else:
        break

print(hour)

File: codeforces/379A/go/379a.go
package main

import "fmt"

func main() {
    var a, b int

    wentout := 0
    hour := 0

    fmt.Scan(&a, &b)
    for a > 0 {
        hour += a
        wentout += a
        if wentout >= b {
            a = wentout / b
            wentout -= a*b
        } else {
            break
        }
    }

    fmt.Println(hour)
}
